### PR TITLE
Create ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by llar

Ticket: [MTM-58015](https://cumulocity.atlassian.net/browse/MTM-58015)
Original PR: [6462](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6462)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: standard applications descriptions are not translated y2024 (#6462)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Standard application descriptions are not translated
```

File: content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58015
+version: 1018.503.92
+---
+standard applications descriptions are not translated y2024 (#6462)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the context of localizing the Cumulocity IoT platform for different languages, it was discovered that the standard application descriptions were not being translated as expected. This change ensures that the standard application descriptions are now properly translated based on the selected platform language. With this fix, users will see the descriptions of standard applications in their chosen language, providing a more localized and user-friendly experience across the Cumulocity IoT platform.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by pmich-sag — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58015
+version: 1018.503.92
+---
+In the context of localizing the Cumulocity IoT platform for different languages, it was discovered that the standard application descriptions were not being translated as expected. This change ensures that the standard application descriptions are now properly translated based on the selected platform language. With this fix, users will see the descriptions of standard applications in their chosen language, providing a more localized and user-friendly experience across the Cumulocity IoT platform.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md:17]:
```suggestion
In the context of localizing the Cumulocity IoT platform for different languages, it was discovered that the standard application descriptions were not being translated as expected. This change ensures that the standard application descriptions are now properly translated based on the selected platform language.
```

File: content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Standard application descriptions are not translated
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1018-503-92-standard-applications-descriptions-are-not-translated-y-2024-6462.md:3]:
```suggestion
title: Standard application descriptions now properly translated
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**